corgi/tasks/management/commands/loaderratadata.py

```diff
@@ -45,6 +48,50 @@ def handle(self, *args, **options) -> None:
                 update_cdn_repo_channels()
             else:
                 update_cdn_repo_channels.delay()
+        elif options["product_variants"]:
+            et = ErrataTool()
+            product_ids = set()
+            for et_variant in CollectorErrataProductVariant.objects.filter(
+                name__in=options["product_variants"]
+            ):
+                product_ids.add(et_variant.product_version.product.et_id)
+            # make sure there is only 1 product id for the names
+            if len(product_ids) > 1:
+                self.stderr.write(
+                    self.style.ERROR(
+                        f"Variants had more than 1 product id. " f"Found: {product_ids}"
```

Comment:
Nitpick: you can combine these two strings. Black messed up the formatting here.

Reply:
fixed.